example_site/src/components/ConfigurationControls.js

```diff
@@ -218,14 +219,16 @@ export default function ConfigurationControls({ isOpen }) {
             opacity: partner === 'custom' ? 1 : 0
           }}
         >
-          <div>
-            <h2>Configuration</h2>
+          <div className="configuration-controls-subsection"> 
+            <div className="setting-title">Configuration</div>
```

Comment:
What is a `setting-title`?

Reply:
Renamed to setting-name

Reply:
It's the `setting` part that is confusing :) 

I think something like `column-title` or `column-name` or `form-name` or something like that would be more descriptive.